### DISCUSSION TITLE
How to change tooltip arrow direction in react-popper?

### DISCUSSION THREAD

**Discussion by aditodkar** [floating-ui/floating-ui#1324] [Q&A]
I have created tooltip using react-popper which uses portals. I have added modifiers with altAxis set to true to prevent overflow (see option.js file in below sandbox). Now in below sandbox tooltip is getting flipped but tooltip arrow should also come down which is not happening in my case. Any idea what I am missing? Please check below sandbox code:
https://codesandbox.io/s/focused-swanson-3b66w?file=/src/option.js:1289-1296

In below screenshot tooltip is getting flipped to top  for option 3 but when there is overflow at bottom but in that case how to adjust tooltip arrow? In below screenshot arrow is still at the top and not bottom.
![tooltipflip](https://user-images.githubusercontent.com/16613832/121857948-bcc48300-cd13-11eb-9372-650d85bd344e.png)

**Comment by steve-taylor** [comment]
The stem placement is contained in `state?.placement`, where `state` is a property in the object returned by `usePopper`.
